elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/OidcActivationProcessor.java

```diff
@@ -48,6 +56,23 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
         OidcConfigService configService = OidcConfigService.getInstance();
         if (configService.isSecureDeployment(deploymentUnit) && configService.isDeploymentConfigured(deploymentUnit)) {
             addOidcAuthDataAndConfig(phaseContext, configService, webMetaData);
+        } else if (deploymentUnit.getAttachment(Attachments.DEPLOYMENT_ROOT) != null) {
+            try {
```

Comment:
Would be good to use `try (InputStream is = ...) {` here.

Reply:
fixed!

Reply:
I don't see this change made, maybe it wasn't pushed? The reason  to do this is so that the input stream will get closed for you.